zipline/algorithm.py

```diff
@@ -403,6 +403,13 @@ def noop(*args, **kwargs):
         if 'data_frequency' in kwargs:
             self.data_frequency = kwargs.pop('data_frequency')
 
+        # Flag for whether or not to compute rolling expected shortfall of the
+        # algorithm. By default this is False because it can be a costly
+        # computation, so only do it if explicitly specified.
+        self.calculate_expected_shortfall = kwargs.pop(
+            'calculate_expected_shortfall', False,
+        )
```

Comment:
I added this in order to avoid doing the `expected_shortfall` calculation unless we specifically pass `True`. Of course on the Quantopian platform this will be `True`.